config/kubernetes/sig-docs/teams.yaml

```diff
@@ -86,6 +86,19 @@ teams:
     - smana
     - yastij
     privacy: closed
+  sig-docs-hi-owners:
+    description: Owners for Hindi content
```

Comment:
Should this be s/Owners/Approvers?

Ref: https://github.com/kubernetes/org/pull/748#issuecomment-486870227

Reply:
Yes it should. @mittalyashu ☝️

Reply:
@mittalyashu can you sort the list alphabetically and change owners to approvers? After that, could you squash the commits into a single commit?

After that this should be good to go. :+1:

Reply:
Sure.